docs/book/src/topics/configuration.md

```diff
@@ -0,0 +1,26 @@
+# Configuration
+
+This page describes how to configure CAPZ installations.
```

Comment:
This is a very broad statement at the root level documentation.  My suggestion was just under the ASO topic area.  At this level, I'd expect much more explanation with all the variables for CAPZ/ASO, what's required/default, and links or included to setup things like multi-tenancy, etc.